packages/ui/src/ui/pages/query-tracker/module/api.ts

```diff
@@ -295,8 +297,17 @@ export function startQuery(
 ): ThunkAction<Promise<{query_id: QueryItemId}>, RootState, any, any> {
     return async (_dispatch, getState) => {
         const state = getState();
+        const isMultipleAco = selectIsMultipleAco(state);
         const {stage, yqlAgentStage} = getQueryTrackerRequestOptions(state);
-        const {query, engine, settings, annotations, files, access_control_objects} = queryInstance;
+        const {
+            query,
+            engine,
+            settings,
+            annotations,
+            files,
+            access_control_objects,
+            access_control_object,
+        } = queryInstance;
```

Comment:
Sending only the necessary data is consistent with best practices in API development and ensures more efficient use of resources. Yes, the code looks simpler, but when working with the API, we need to follow the following guidelines::

1) Data Consistency: Sending only the necessary data can help ensure data consistency in the system. If you send only the necessary data, then other parts of the system can be sure that they receive only the data they need, and that they will not receive unnecessary or irrelevant data.
2) Data Reduction: Sending only the necessary data reduces the amount of data that needs to be transmitted over the network. This can reduce data transfer time and reduce the load on network resources.